README.md

```diff
@@ -27,9 +27,9 @@ The toolkit provides five separate packages.   See the docs for each action.
 
 Actions are units of work which can either run in a container or on the host machine.
 
-[Choosing an action type](docs/action-types.md): Outlines the differences and why you would want to create a host or a container based action.
+[Choosing an action type](docs/action-types.md): Outlines the differences and why you would want to create a JavaScript or a container based action.
```

Comment:
Should we make it explicit here that we recommend Typescript to make life easier?

Reply:
I think the next block covers that.  Choosing an action type is orthogonal to dev tooling chosen.  Case in point is our next action / in official docs is a topic on creating an action from scratch (hello world) in pure javascript.  This doc strictly covers javascript vs container and why you would do one or the other.